source/contracts/reporting/Market.sol

```diff
@@ -233,24 +225,23 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
             return false;
         }
 
-        if (getUniverse().getForkingMarket() == this) {
+        if (getForkingMarket() == this) {
             tentativeWinningPayoutDistributionHash = getWinningPayoutDistributionHashFromFork();
         }
 
-        require(tentativeWinningPayoutDistributionHash != bytes32(0));
-
         finalPayoutDistributionHash = tentativeWinningPayoutDistributionHash;
         finalizationTime = block.timestamp;
         transferIncorrectDisputeBondsToWinningReportingToken();
+        // The validity bond is paid to the owner in any indeterminate outcome and the reporting window otherwise
```

Comment:
This comment is backward.  Owner gets the bond if the final outcome is valid, reporting window gets it if the final outcome is invalid.

Reply:
Ah yup my bad